packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -28,3 +28,11 @@ export * from './lib/directives/amplify-slot.directive';
  */
 export * from './lib/services/authenticator-context.service';
 export * from './lib/services/state-machine.service';
+
+/**
+ * Translations
+ */
+
+import { I18n } from 'aws-amplify';
+import { translations } from '@aws-amplify/ui';
+I18n.putVocabularies(translations);
```

Comment:
This gets run at import instead of when the Authenticator loads.

Reply:
We need to be careful about this because it means there are side effects and no longer tree-shakable. 

Angular had specific issues with this.

Reply:
The other way is we tell the customer to run the `I8n.putVocabulatries(translations)` themselves What do you think @ericclemmons

Reply:
@ErikCH You know, I think you found the reason where our good intentions (automatically translated) create problems.

`I18n` is a naive utility that doesn't really have a way for us to say, "Here are some defaults, if you don't already have them".  Which is what we want.

**My recommendation for resolution is the following**:

- [ ] Remove `I18n.putVocabularies` from our implementations to fix this bug.
- [ ] Update https://ui.docs.amplify.aws/ui/components/authenticator?platform=react#internationalization-i18n with instructions on how to add opt-in to translations for customer apps.
- [ ] Update the `/ui/components/authenticator/i18n` examples to match the docs

This allows them to choose when the translations are added, avoiding race-conditions.

Reply:
👍

Reply:
I pushed up the changes